shimmingtoolbox/cli/mask.py

```diff
@@ -303,6 +304,41 @@ def sct(fname_input, fname_output, contrast, centerline, file_centerline, brain,
     return fname_output
 
 
+@mask_cli.command(context_settings=CONTEXT_SETTINGS,
+                  help="Create a mask to shim single voxel MRS. "
+                       "Voxel position and size can be directly given or these info can be read "
+                       "from the siemens raw-data. "
+                       "The mask is stored by default under the name 'mask_mrs.nii.gz' in the output "
+                       "folder. Return an output nifti file to be used as a mask for MRS shimming.")
+@click.option('-i', '--input', 'fname_input', type=click.Path(), required=True,
+              help="Input path of the fieldmap to be shimmed.")
+@click.option('-r', '--raw_data', type=click.Path(),
+              help="Input path of the of the siemens raw-data (supported extention .rda)")
+@click.option('-o', '--output', type=click.Path(), default=os.path.join(os.curdir, 'mask_mrs.nii.gz'),
+              show_default=True, help="Name of the output mask. Supported extensions are .nii or .nii.gz. (default: "
+              "(os.curdir, 'mask_mrs.nii.gz'))")
```

Comment:
No need to tell what the default is in the description. You already see it with the option "show_default"

```suggestion
              show_default=True, help="Name of the output mask. Supported extensions are .nii or .nii.gz.")
```